test/e2e/flask/snaps/preinstalled-example.spec.ts

```diff
@@ -37,7 +37,7 @@ describe('Pre-install example', function () {
         await testSnaps.openPage();
         await testSnaps.scrollAndClickButton('connectPreinstalledButton');
         await driver.switchToWindowWithTitle(WINDOW_TITLES.Dialog);
-        await snapInstall.clickConfirmButton();
```

Comment:
naming is incorrect, as we were using the same selector for 3 different buttons (Connect, Confirm and Ok) -- see below